package.json

```diff
@@ -3,9 +3,11 @@
   "version": "2.5.0",
   "description": "Local Edge Side Includes parser",
   "main": "index.js",
+  "type": "module",
   "scripts": {
     "test": "mocha",
-    "posttest": "eslint . --cache"
+    "posttest": "eslint . --cache",
+    "dist": "./node_modules/.bin/rollup index.js --format cjs --file index.cjs && ./node_modules/.bin/rollup ./lib/voidElements.js --format cjs --file ./lib/voidElements.cjs"
```

Comment:
kanske bättre om det görs i samband med `npm publish`. finns det inte typ `npm prepublish` eller `npm pack`?

Reply:
isåfall kanske det inte behövs något `.gitignore`? 🤔

Reply:
När jag gjort liknande för ett open-source lib så har jag lagt det i prepack, vet inte om det är best practice men det funkar fint och automagiskt.